cmd/monaco/download/download_command.go

```diff
@@ -84,6 +84,13 @@ func GetDownloadCommand(fs afero.Fs, command Command) (cmd *cobra.Command) {
 	cmd.MarkFlagsMutuallyExclusive("api", "only-apis", "only-settings")
 	cmd.MarkFlagsMutuallyExclusive("only-apis", "only-settings")
 
+	if featureflags.AutomationResources().Enabled() {
```

Comment:
Is the addition of only-automation needed for this PR, or was there some miscommunication with @warber, who also opened a PR for it? https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1003/files

I agree with placing this under the FF - could this change just be added to the other PR which also extends the tests, and be dropped from this changeset (if not needed for the changes here)?

Reply:
It was convenient for me to implement it to make manual testing easier.

I didn't see that @warber also did it.

Reply:
closed #1003 in favor of this